api/rpp.h

```diff
@@ -50,8 +50,13 @@ typedef hipStream_t rppAcceleratorQueue_t;
 
 #endif
 
+#if __cplusplus >= 201703L && __has_include(<filesystem>)
```

Comment:
@r-abishek : Do we need to include filesystem in the main header. We shouldn't include here. Can we move this to an internal file